fe/fe-core/src/main/java/org/apache/doris/external/elasticsearch/QueryBuilders.java

```diff
@@ -172,19 +183,65 @@ abstract static class QueryBuilder {
          * @throws IOException if IO error occurred
          */
         abstract void toJson(JsonGenerator out) throws IOException;
+
+        /**
+         * Convert query to JSON format and catch error.
+         **/
+        public String toJson() {
+            StringWriter writer = new StringWriter();
+            try {
+                JsonGenerator gen = mapper.getFactory().createGenerator(writer);
+                this.toJson(gen);
+                gen.flush();
+                gen.close();
+            } catch (IOException e) {
+                LOG.warn("QueryBuilder toJson error");
```

Comment:
The error should print to warn log.

Reply:
Step one, in order to be compatible, fe generate dsl can not affect exist logic, what do you think?